src/test/javascript/integration/webhook_error_channel_spec.js

```diff
@@ -109,6 +117,7 @@ describe(__filename, () => {
 
     it('waits for the webhook to give up', async () => {
         const clause = (response) => {
+            console.log('response.body', response.body.errors);
```

Comment:
Two things.

a) The string prefix should read `response.body.errors`
b) This line should be after the next line so we can use the `errors` constant that has already been safely extracted

Reply:
good point